### PR TITLE
Reorganized chemistry reagent recipes

### PR DESCRIPTION
**Pull request by N3X15**
- Chemistry recipes dumped into their own categorized files.
- Recipes use a list of results rather than scalars (permits producing byproducts)
- Absolute pathing
- required_temperature changed into min_temperature/max_temperature.

### CODE REVIEW

**Comment by wwjnc** [comment]
Based Eggman.

File: code/modules/reagents/recipes/food/food.dm [hunk 1]
```diff
@@ -0,0 +1,96 @@
+
+/datum/chemical_reaction/tofu
+	name = "Tofu"
+	id = "tofu"
+	required_reagents = list("soymilk" = 10)
+	required_catalysts = list("enzyme" = 5)
+	results = list(null = 1)
```
Comment (N3X15) [inline code/modules/reagents/recipes/food/food.dm]: aaaa

**Comment by SkowronX** [comment]
Master plan.

**Comment by N3X15** [comment]
There, fixed.

**Comment by Probe1** [comment]
He's the parser we need, not the one we deserve

**Pull request merged**